telegram/bot.py

```diff
@@ -1180,37 +1278,37 @@ def edit_message_text(self,
                           reply_markup=None,
                           timeout=None,
                           **kwargs):
-        """Use this method to edit text messages sent by the bot or via the bot (for inline bots).
+        """
+        Use this method to edit text and game messages sent by the bot or via the bot (for inline
+        bots).
 
         Args:
-            text (str): New text of the message.
-            chat_id (Optional[int|str]): Required if inline_message_id is not specified. Unique
-                identifier for the target chat or username of the target channel (in the format
-                @channelusername).
-            message_id (Optional[int]): Required if inline_message_id is not specified. Unique
-                identifier of the sent message.
-            inline_message_id (Optional[str]): Required if chat_id and message_id are not
+            chat_id (:obj:`int` | :obj:`float`): Unique identifier for the target chat or username
+                of the target`channel (in the format @channelusername).
+            message_id (:obj:`int`, optional): Required if inline_message_id is not specified.
+                Identifier of the sent message.
+            inline_message_id (:obj:`str`, optional): Required if chat_id and message_id are not
                 specified. Identifier of the inline message.
-            parse_mode (:class:`telegram.ParseMode`|str): Send Markdown or HTML, if you want
-                Telegram apps to show bold, italic, fixed-width text or inline URLs in your bot's
-                message.
-            disable_web_page_preview (bool): Disables link previews for links in this message.
-            reply_markup (Optional[:class:`telegram.ReplyMarkup`]): Additional interface options. A
+            text (:obj:`str`): New text of the message.
+            parse_mode (:class:`telegram.ParseMode` | :obj:`str`): Send Markdown or HTML, if you
```

Comment:
it isn;t but for clarity I think this is the best thing to do. Other option would be something like (constant from telegram.parsemode | str) which is damn ugly